utility/osp_deploy.sh

```diff
@@ -21,29 +23,47 @@ sudo yum install -y libguestfs-tools-c wget
 
 if [ -n "$1" -a -f "$1" ]; then
     mkdir newpkgs
-    echo "using local rpm file: $1"
-    cp "$1" newpkgs/
+    echo "using local rpm files: $@"
+    cp $@ newpkgs/
 else
+    [ -d newpkgs ] || mkdir newpkgs
+    cd newpkgs
+
     release='latest'
     [ -n "$1" ] && release="tags/r$1"
 
-    rel_info="$(curl -s https://api.github.com/repos/emc-openstack/storops/releases/$release)"
-    if [[ $rel_info == *"\"message\": \"Not Found\""* ]]; then
-        echo "invalid release."
-        exit 1
-    fi
+    git_repos="emc-openstack/storops/releases/$release \
+               peter-wangxu/persist-queue/releases/tags/v0.2.3 \
+               jealous/cachez/releases/tags/r0.1.2 \
+               jealous/retryz/releases/tags/r0.1.9"
 
-    download_url=$(echo "$rel_info" | grep -oh "browser_download_url\": \".*\"" | cut -d' ' -f2 | cut -d'"' -f2)
-    echo "$download_url"
-    if [ -z "$download_url" ]; then
-        echo "no rpm url found."
-        exit 1
-    fi
+    wget_urls="http://cbs.centos.org/kojifiles/packages/python-bitmath/1.3.1/1.el7/noarch/python2-bitmath-1.3.1-1.el7.noarch.rpm \
+               https://github.com/emc-openstack/naviseccli/raw/master/NaviCLI-Linux-64-x86-en_US-7.33.9.1.55-1.x86_64.rpm"
+
+    for repo in ${git_repos}; do
+
+        rel_info="$(curl -s https://api.github.com/repos/$repo)"
+        if [[ ${rel_info} == *"\"message\": \"Not Found\""* ]]; then
+            echo "invalid release."
+            exit 1
+        fi
+
+        download_url=$(echo "$rel_info" | grep -oh "browser_download_url\": \".*\"" | grep ".rpm" | cut -d' ' -f2 | cut -d'"' -f2)
+        echo "${download_url}"
+        if [ -z "$download_url" ]; then
+            echo "no rpm url found."
+            exit 1
+        fi
+
+        echo "downloading rpm from: $download_url"
+        wget ${download_url}
+    done
+
+    for url in ${wget_urls}; do
+        echo "downloading rpm from: $url"
+        wget ${url}
```

Comment:
The comment is for the last line of this file.
Red Hat suggest to change
yum localinstall -y /root/newpkgs/*rpm
to
yum localinstall -y /root/newpkgs/*.rpm

Add "." before "rpm"